.pre-commit-config.yaml

```
@@ -20,4 +20,4 @@ repos:
         # these are errors that will be ignored by flake8
         # check out their meaning here
         # https://flake8.pycqa.org/en/latest/user/error-codes.html
-        - "--ignore=E501,E203"
+        - "--ignore=E501,E203,W503"
```

what is this error being ignored?